src/integrations/buttons.js

```diff
@@ -140,6 +141,29 @@ function addHostCopyNetworksButton () {
   })
 }
 
+/**
+ * "Manage Storage Connections. Enabled when exactly 1 iSCSI domain is selected
+ */
+function addManageStorageConnectionsButton () {
+  getPluginApi().addMenuPlaceActionButton(entityTypes.storage, msg.storageConnectionsManageButton(), {
+    onClick: function ([selectedDomain]) {
+      showStorageConnectionsModal(selectedDomain.id)
+    },
+
+    isEnabled: function (selectedDomains) {
+      return (
+        selectedDomains.length === 1 &&
+        selectedDomains[0] &&
+        selectedDomains[0].type &&
```

Comment:
how can this happen?

Reply:
Just in case the server returns incorrect object - so that it doesn't access null's fields and throws exception